docs/architecture/platform-versions.md

```diff
@@ -91,11 +91,15 @@ History:
 * We dropped iOS 10 support in 2020-10. It was 0.3% of iOS users who
   tried Zulip, and we wanted to use a feature introduced in iOS 11,
   called "named colors".
+* We [dropped iOS 11 support][] in 2021-04. It was 0.1% of iOS users
+  who tried Zulip. We started iOS 12 support at 12.1 because Xcode's
+  dropdown for "Deployment Target" didn't have a 12.0.
```

Comment:
Could you add a comment here like:

```
<!-- When updating this, please update `docs/developer-guide.md` as well. -->
```

Alternatively, we might want to consider just having fewer places where we keep this information :) I'd be fine with removing it from `docs/developer-guide.md` altogether, or keeping a link to this doc but not explicitly writing down the versions.